ql/src/test/org/apache/hadoop/hive/ql/TestTxnLoadData.java

```diff
@@ -235,18 +235,18 @@ private void loadData(boolean isVectorized) throws Exception {
     runStatementOnDriver("export table Tstage to '" + getWarehouseDir() +"/2'");
     runStatementOnDriver("load data inpath '" + getWarehouseDir() + "/2/data' overwrite into table T");
     String[][] expected3 = new String[][] {
-        {"{\"writeid\":5,\"bucketid\":536870912,\"rowid\":0}\t5\t6", "t/base_0000005/000000_0"},
```

Comment:
Check line 222
`runStatementOnDriver("alter table T compact 'major'")`
Since stats gathering no longer increases the write id, (from now on) neither compactions do